src/components/AllArticles.jsx

```diff
@@ -7,17 +7,20 @@ import TopicsList from "./TopicsList";
 const AllArticles = ( )=> {
     const [articles, setArtcicles] = useState ([]);
     const [isLoading, setIsLoading] = useState(true)
+    const [sortBy, setSortBy] =useState('created_at')
+    const [orderBy, setOrderBy] =useState('asc')
```

Comment:
It might be better to set the default state as desc - so most recent articles are show first.